teanga/corpus.py

```diff
@@ -687,6 +688,24 @@ def _to_json(self, writer):
                            for layer_id in doc.layers}
         json.dump(dct, writer)
 
+    def to_tcf(self, path:str):
+        """Write the corpus to a TCF file.
+
+        Parameters:
+        -----------
+
+        path: str
+            The path to the TCF file.
```

Comment:
the docs are not in the mkdocs format,  Parameters:
should be replace by args and "-------" removed. 

        Args:
            path: str
                    The path to the TCF file.